.changeset/odd-months-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Prefer JSON responses when returning errors if accept header is _/_
```

Comment:
```suggestion
Prefer JSON responses when returning errors if accept header is \*/\*
```

I think Prettier got its hands on this and changed `*/*` to `_/_`, because an emphasized forward slash is definitely what you wanted.

Reply:
fucking hell computers are idiots

Reply:
```suggestion
Prefer JSON responses when returning errors if accept header is `*/*`
```